pkgs/development/node-packages/node-packages.nix

```diff
@@ -1363,6 +1363,24 @@ let
         sha1 = "cbc4b9a68981bf0b501ccd06a9058acd65309bf7";
       };
     };
+    "@discordjs/opus-0.1.0" = {
```

Comment:
this is probably breaking without the native opus lib in the inputs. But we can keep it like so until someone actually wants to use it :)